plugin.video.jwtv-unofficial/resources/language/English/strings.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<strings>
```

Comment:
Can you use the new .po format for language files? Also please use the new folder layout. See here an example: https://github.com/xbmc/generator-kodi-addon/tree/master/generators/app/templates/resources/language